include/guestbook.db.inc

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+* @file
+* Contains db callbacks and functions for guestbook module.
+*/
+
+/**
+ * Returns data from the guestbook database.
+ *
+ * @return array
+ *   A data array containing complete information about each database entry.
+ */
+function guestbook_read_db($table = 'guestbook_drupal', $order = 'id', $direction = 'DESC', $limit = '10') {
+  $rows = array();
+  $query = db_select($table, 't')
+    ->extend('PagerDefault')
+    ->fields('t', array('id', 'uid', 'sid', 'date', 'message'))
+    ->orderBy($order, $direction)
+    ->limit($limit)
+    ->execute();
+
+  while ($value = $query->fetchAssoc()) {
+    $rows[] = array(
+      'id' => $value['id'],
+      'uid' => $value['uid'],
+      'sid' => $value['sid'],
+      'date' => $value['date'],
+      'message' => check_plain($value['message']),
+      'user_name' => guestbook_name_by_id($value['uid']),
+      'formatted_date' => format_date($value['date'], $type = 'medium',
```

Comment:
@PavelSPN you don't need set `$type = 'medium'` and other args since they are default, please remove them